FOS.Models/Constants/Constants.cs

```diff
@@ -255,9 +255,38 @@ public static class SqlColumnNames
             public const string GenderName = "Gender_Name";
             public const string DateofBirth = "Prospect_DateofBirth";
             public const string MobileNumber = "Mobile_Number";
+            public const string MobileNumberr = "Mobile Number";
```

Comment:
Let s not have duplicated. Please correct the SP if required. Please take it as an exercise to remove the duplicates